### DISCUSSION TITLE
gym.error.NameNotFound: Environment BreakoutDeterministic doesn't exist.

### DISCUSSION THREAD

**Discussion by JiangJingru** [BindsNET/bindsnet#653] [Q&A]
gym.error.NameNotFound: Environment BreakoutDeterministic doesn't exist.

**Comment by Hananel-Hazan** [comment]
Thanks for reporting this. 
Can you provide more details to replicate this error?

**Reply by Hananel-Hazan** [reply]
Please try PR #655 and let me know if its solve the issue
